src/ape_test/provider.py

```diff
@@ -133,8 +134,15 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
         receipt = self.get_receipt(
             txn_hash.hex(), required_confirmations=txn.required_confirmations or 0
         )
-        if txn.gas_limit is not None and receipt.ran_out_of_gas:
-            raise OutOfGasError()
+
+        if receipt.failed:
```

Comment:
Here is my hacky fix, but perhaps this is a bug that should be resolved in EthTester